android/src/com/frostwire/android/gui/activities/ToggleAllSearchEnginesPreference.java

```diff
@@ -60,15 +60,15 @@ public CheckBox getCheckbox() {
     @Override
     protected View onCreateView(ViewGroup parent) {
         View view = super.onCreateView(parent);
-        final TextView titleView = (TextView) view.findViewById(R.id.checkbox_preference_title);
+        final TextView titleView = (TextView) view.findViewById(R.id.view_preference_checkbox_header_title);
         titleView.setText(getTitle());
-        view.findViewById(R.id.checkbox_preference_summary).setVisibility(View.INVISIBLE);
 
-        checkbox = (CheckBox) view.findViewById(R.id.checkbox_preference_checkbox);
+        checkbox = (CheckBox) view.findViewById(R.id.view_preference_checkbox_header_checkbox);
         checkbox.setVisibility(View.VISIBLE);
         checkbox.setClickable(true);
         checkbox.setChecked(areAllEnginesChecked());
 
+
```

Comment:
it does. And as to the name change for the CheckBoxPreference to seeding - yes it was used only for seeding. For everything else you could just use a LinearLayout with "@+android:id/widget_frame" and if you used CheckBoxPreference or SwitchPreference etc it would just place the correct thing into that layout. 
The CheckboxSeedingPreference uses a checkbox specifically so it could not be used on all other preferences and you use the id of that checkbox specifically for the logic related to seeding.